utils/apiRequests.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+export const getArticlesList = () => {
+    return axios.get(`https://nc-news-eph7.onrender.com/api/articles/`)
```

Comment:
Great that you've extracted this functionality. I recommend using baseURL with axios 'create'.